tests/item_test.cpp

```diff
@@ -1930,7 +1923,8 @@ TEST_CASE( "item_material_density_blacklist_is_pruned", "[item][!mayfail]" )
         }
         const std::map<material_id, int> &mats = target.made_of();
         if( !mats.empty() ) {
-            INFO( bad.str() );
+            INFO( string_format( "%s had its density fixed, remove it from the known_bad list in tests\\item_test.cpp",
```

Comment:
GitHub displays unix-style paths.
```suggestion
            INFO( string_format( "%s had its density fixed, remove it from the known_bad list in tests/item_test.cpp",
```